Description: fixing some spelling errors
Author: Nicolas HOUDELOT (nicolas@demosdebs.org)
Last-Update: 2020-04-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/source/build/src/polymost.cpp
+++ b/source/build/src/polymost.cpp
@@ -9904,7 +9904,7 @@
     if (parm->numparms != 1)
     {
         LOG_F(INFO, "Current texturing mode is %s", glfiltermodes[gltexfiltermode].name);
-        LOG_F(INFO, "  Vaild modes are:");
+        LOG_F(INFO, "  Valid modes are:");
         for (m = 0; m < NUMGLFILTERMODES; m++)
             LOG_F(INFO, "     %d - %s", m, glfiltermodes[m].name);
 
--- a/source/build/src/animvpx.cpp
+++ b/source/build/src/animvpx.cpp
@@ -193,7 +193,7 @@
 
 const char *animvpx_nextpic_errmsg[] = {
     "All OK",
-    "INTERNAL ERROR, animvpx_codec_ctx not initalized!",
+    "INTERNAL ERROR, animvpx_codec_ctx not initialized!",
     "OUT OF MEMORY",
     "couldn't read whole frame",
     "decoder error, couldn't decode frame",
--- a/source/duke3d/src/astub.cpp
+++ b/source/duke3d/src/astub.cpp
@@ -7762,7 +7762,7 @@
             }
             else
             {
-                printmessage16("No sprite higlighted, cannot cycle linking sprites.");
+                printmessage16("No sprite highlighted, cannot cycle linking sprites.");
             }
         }
         else if (wallsprite==0)
--- a/source/build/src/config.cpp
+++ b/source/build/src/config.cpp
@@ -484,7 +484,7 @@
              "; Reverse meaning of Q and W keys in side view mode\n"
              "sideview_reversehorizrot = %d\n"
              "\n"
-             "; Revert CTRL for tile selction\n"
+             "; Revert CTRL for tile selection\n"
              ";   0 - WHEEL:scrolling, CTRL+WHEEL:zooming\n"
              ";   1 - CTRL+WHEEL:scrolling, WHEEL:zooming\n"
              "revertCTRL = %d\n"
